target/classes/index.jelly

```diff
@@ -0,0 +1,5 @@
+<?jelly escape-by-default='true'?>
```

Comment:
why do we need all these file (this file and all others) in _target_ folder? how is plugin installing? does it need all these files or build output is one file?

Reply:
this jelly file is the main file, the text written here is visible text when we update the hpi file